### ISSUE TITLE
Should log determinant be adjusted for some of the flows in affine.py?

### ISSUE THREAD

**Issue by francisa6** [Information-Fusion-Lab-Umass/NuX#10]
For example:

class CenterAndScale(Flow):

  def __init__(self, s):
    assert s < 1.0
    self.s = s
    self.b = (1.0 - s)/2

  def get_params(self):
    return {}

  def __call__(self, x, params=None, inverse=False, **kwargs):
    s = jnp.broadcast_to(self.s, x.shape)
    log_s = jnp.log(s)

    if inverse == False:
      z = x*s + self.b
    else:
      z = (x - self.b)/s

    sum_axes = util.last_axes(x.shape[1:])
    log_det = log_s.sum(axis=sum_axes)
    return z, log_det

The log_det for the inverse should be -log_det.
